DataFormats/METReco/interface/HcalPhase1FlagLabels.h

```diff
@@ -20,7 +20,8 @@ namespace HcalPhase1FlagLabels
         /*  0 */ HFLongShort = HcalCaloFlagLabels::HFLongShort,
         /*  3 */ HFS8S1Ratio = HcalCaloFlagLabels::HFS8S1Ratio,
         /*  4 */ HFPET = HcalCaloFlagLabels::HFPET,
-        /*  5 */ HFSignalAsymmetry = 5
+        /*  5 */ HFSignalAsymmetry = 5,
```

Comment:
We keep HcalCaloFlagLabels untouched in case somebody wants to reprocess pre-Phase-1 data. The hardware was different then (no PMTs with dual anode readout, no hfprereco module in the sequence, etc), and the meaning of these bits was different as well. In particular, we had HSCP_ExpFit=5 and HBHETimingTrustBits=6. For Phase-1, the HCAL Noise Group performed a review of all status bits and decided that some of them can be repurposed.